src/wfs/src/test/java/org/geoserver/wfs/xml/GMLOutputFormatTest.java

```diff
@@ -60,6 +62,35 @@ public void testGML2() throws Exception {
         assertNull(getFirstElementByTagName(dom, "gml:exterior"));
     }
 
+    @Test
+    public void testGML2CoordinatesFormatting() throws Exception {
+        enableCoordinatesFormatting();
```

Comment:
Would be best to restore the config the way it was after the test, you could use a @After method to do that.